Core/src/com/serotonin/m2m2/db/DatabaseProxy.java

```diff
@@ -239,12 +237,14 @@ default void initializeCoreDatabase(DSLContext context) {
                 .get(permissions.id);
 
         if (Common.envProps.getBoolean("initialize.admin.create")) {
-            long created = System.currentTimeMillis();
+            long createdTs = System.currentTimeMillis();
+            String defaultPassword = Common.envProps.getProperty("initialize.admin.password");
+            long passwordChangeTs = defaultPassword.equals("admin") ? createdTs : System.currentTimeMillis();
```

Comment:
Can you change this to use `createdTs + 1` instead of `System.currentTimeMillis()`